XCast/XCast.json

```diff
@@ -214,7 +214,7 @@
             }
         },
         "setStandbyBehavior":{
-            "summary": "Sets the expected xcast behavior in standby mode",
+            "summary": "Sets the expected xcast behavior in standby mode.\n  \n### Events \n\n No Events",
```

Comment:
This API allows application to override controls the xcast behavior in standby mode. Default behavior in STANDBY mode is disable so client device can not discover the server. When STANDBY behavior is active client device could discover the server.

Reply:
Updated changes as per your review comments